### ISSUE TITLE
Filter works in other adblockers but not uBO

### ISSUE THREAD

**Issue by mimecry** [uBlockOrigin/uAssets#3283]
### URL(s) where the issue occurs

`reddit.com`

### Describe the issue

By default, Reddit uses its new favicon with orange background. A method to revert back to its classic blue grey background is to add the following filter to your adblocker: `redditstatic.com/desktop2x/img/favicon/*`. This works in adblockers such as Adblock Plus and Adblock Latitude, but not uBlock Origin.

### Screenshot(s)

![orange favicon](https://i.imgur.com/eoFF12z.png)
what it looks like with the filter added

![blue&grey favicon](https://i.imgur.com/QU2EpKJ.png)
what it should look like (and does on other adblockers) with the filter added 

### Versions

- Browser/version: Pale Moon 28.0.0
- uBlock Origin version: uBO Legacy 1.16.4.4

### Settings

- No changes, tested on a fresh profile with a new uBO install

### Notes

None

**Comment by uBlock-user** [comment]
> A method to revert back to its classic blue grey background is to add the following filter to your adblocker: redditstatic.com/desktop2x/img/favicon/*.

That doesn't happen on Chromium even with the filter, you're being served an old layout of the page probably because you're on Pale Moon.

**Comment by uBlock-user** [comment]
> Request context is behind-the-scene.

Not on Chromium and Firefox

**Comment by gwarser** [comment]
Yep, I cannot get old favicon too. And https://bugzilla.mozilla.org/show_bug.cgi?id=1433700

**Comment by mimecry** [comment]
you guys are right, when the filter is added to ABP or uBO on Slimjet (Chrome-based browser), the favicon shows up as blank and the old favicon does not return.

has anyone tried to reproduce my report on Pale Moon? i know at least one poster on the PM forums has experienced the same

**Comment by uBlock-user** [comment]
> has anyone tried to reproduce my report on Pale Moon

Pale Moon is not officially supported, so reproduction is always done on Chrome/Firefox.
